samples/dect/dect_phy/dect_shell/src/dect/mac/dect_phy_mac_shell.c

```diff
@@ -0,0 +1,553 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <stdbool.h>
+#include <stdint.h>
+#include <string.h>
+#include <stdlib.h>
+
+#include <zephyr/kernel.h>
+
+#include <zephyr/shell/shell.h>
+#include <getopt.h>
+
+#include "desh_print.h"
+
+#include "dect_common.h"
+#include "dect_common_utils.h"
+#include "dect_common_settings.h"
+
+#include "dect_phy_ctrl.h"
+
+#include "dect_phy_mac_common.h"
+#include "dect_phy_mac_cluster_beacon.h"
+#include "dect_phy_mac_nbr.h"
+#include "dect_phy_mac_ctrl.h"
+
+/**************************************************************************************************/
+
+static int dect_phy_mac_cmd_print_help(const struct shell *shell, size_t argc, char **argv)
+{
+	int ret = 1;
+
+	if (argc > 1) {
+		shell_error(shell, "%s: subcommand not found", argv[1]);
+		ret = -1;
+	}
+
+	shell_help(shell);
+
+	return ret;
+}
+
+static int dect_phy_mac_cmd(const struct shell *shell, size_t argc, char **argv)
+{
+	return dect_phy_mac_cmd_print_help(shell, argc, argv);
+}
+
+/**************************************************************************************************/
+
+static const char dect_phy_mac_beacon_scan_usage_str[] =
+	"Usage: dect mac beacon_scan [-c <channel>] [<other_options>]\n"
+	"Options:\n"
+	"  -c <integer>,                        Channel nbr to be scanned for a beacon.\n"
+	"                                       Ranges: band #1: 1657-1677, band #2 1680-1700,\n"
+	"                                       band #9 1691-1711. Zero value: all in a set band.\n"
+	"                                       Default: 1665.\n"
+	"  -t, --c_scan_time <integer>,         Scanning duration in seconds (default: 4 "
```

Comment:
There are still several options with `c_` and `b_` prefixes in different commands. I think you could consider removing the prefixes from those as well. For example, short option `-p` is used for TX power for both beacon and client, but the long options are `--b_tx_pwr` and `--c_tx_pwr`. I don't see the need for this kind of separation.

Reply:
Good point. I'll go through also these.

Reply:
Good point. I'll go through also these.